evm/utils/headers.py

```diff
@@ -65,3 +69,29 @@ def compute_gas_limit(parent_header, gas_limit_floor):
         return parent_header.gas_limit + decay
     else:
         return gas_limit
+
+
+def generate_header_from_parent_header(
+        compute_difficulty,
+        parent_header,
+        timestamp,
+        coinbase=b'\x35' * 20,
```

Comment:
Where does `b'\x35` * 20` come from and can we get it moved into a constant variable somewhere?

Reply:
I'll remove it.